cmd/gateway-azure-anonymous.go

```diff
@@ -29,18 +29,59 @@ import (
 	"github.com/Azure/azure-sdk-for-go/storage"
 )
 
+// Make anonymous HTTP request to azure endpoint.
+func azureAnonRequest(verb, urlStr string, header http.Header) (*http.Response, error) {
```

Comment:
We had decided not to do mock tests. i.e we should not start a mock server and test instead should use the actual backend in functional tests in mint to cover cases like these.